app/component/map/tile-layer/TileLayerContainer.js

```diff
@@ -217,7 +217,7 @@ class TileLayerContainer extends GridLayer {
     );
 
     if (typeof this.state.selectableTargets !== 'undefined') {
-      if (this.state.selectableTargets.length === 1) {
+      if (this.state.selectableTargets.length >= 1) {
```

Comment:
Don't think this is the cause neither we should fix it that way. In line [335](https://github.com/mfdz/digitransit-ui/blob/e44211b7bdc5a2aec35d1ca8818b347eecf0b22f/app/component/map/tile-layer/TileLayerContainer.js#L355) the case for multiple selectable features is already handled (e.g. multiple stops at the same place), so we would break functionality